src/EXTRA-FIX/fix_ttm_mod.cpp

```diff
@@ -348,14 +335,14 @@ void FixTTMMod::post_force(int /*vflag*/)
           if (left_x == -1) left_x = nxgrid - 1;
           if (left_y == -1) left_y = nygrid - 1;
           if (left_z == -1) left_z = nzgrid - 1;
-          double T_i = T_electron[ix][iy][iz];
-          double T_ir = T_electron[right_x][iy][iz];
-          double T_iu = T_electron[ix][right_y][iz];
-          double T_if = T_electron[ix][iy][right_z];
-          double C_i = el_properties(T_electron[ix][iy][iz]).el_heat_capacity;
-          double C_ir = el_properties(T_electron[right_x][iy][iz]).el_heat_capacity;
-          double C_iu = el_properties(T_electron[ix][right_y][iz]).el_heat_capacity;
-          double C_if = el_properties(T_electron[ix][iy][right_z]).el_heat_capacity;
+          double T_i = T_electron[iz][iy][ix];
+          double T_ir = T_electron[right_z][iy][ix];
```

Comment:
@sjplimp are you sure that these changes are correct? `T_ir` changes from "right_x" to "right_z" and `T_if` the other way around.